backend/src/migrations/20240210124346_create_tables.ts

```diff
@@ -0,0 +1,205 @@
+        table.string(DatabaseColumnName.EMAIL).unique().notNullable();
+        table.string(DatabaseColumnName.USERNAME).notNullable();
+        table.text(DatabaseColumnName.PASSWORD_HASH).notNullable();
+        table.text(DatabaseColumnName.PASSWORD_SALT).notNullable();
+        table
+            .dateTime(DatabaseColumnName.CREATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+        table
+            .dateTime(DatabaseColumnName.UPDATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+    });
+    await knex.schema.createTable(DatabaseTableName.PROFILE, table => {
+        table.uuid(DatabaseColumnName.ID).primary();
+        table.string(DatabaseColumnName.FIRST_NAME).notNullable();
+        table.string(DatabaseColumnName.LAST_NAME).notNullable();
+        table.string(DatabaseColumnName.AVATAR).notNullable();
+        table
+            .dateTime(DatabaseColumnName.CREATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+        table
+            .dateTime(DatabaseColumnName.UPDATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+    });
+    await knex.schema.createTable(
+        DatabaseTableName.PERSONAL_INFORMATION,
+        (table) => {
+            table.uuid(DatabaseColumnName.ID).primary();
+            table.string(DatabaseColumnName.PROFESSION).notNullable();
+            table.string(DatabaseColumnName.ADDRESS).notNullable();
+            table.string(DatabaseColumnName.CITY).notNullable();
+            table.string(DatabaseColumnName.STATE).notNullable();
+            table
+                .dateTime(DatabaseColumnName.CREATED_AT)
+                .notNullable()
+                .defaultTo(knex.fn.now());
+            table
+                .dateTime(DatabaseColumnName.UPDATED_AT)
+                .notNullable()
+                .defaultTo(knex.fn.now());
+        },
+    );
+    await knex.schema.createTable(DatabaseTableName.EXPERIENCE, (table) => {
+        table.uuid(DatabaseColumnName.ID).primary();
+        table.string(DatabaseColumnName.JOB_TITLE).notNullable();
+        table.string(DatabaseColumnName.EMPLOYER).notNullable();
+        table.string(DatabaseColumnName.EMPLOYMENT_TYPE).notNullable();
+        table.date(DatabaseColumnName.START_DATE).notNullable();
+        table.date(DatabaseColumnName.END_DATE).notNullable();
+        table
+            .dateTime(DatabaseColumnName.CREATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+        table
+            .dateTime(DatabaseColumnName.UPDATED_AT)
+            .notNullable()
+            .defaultTo(knex.fn.now());
+    });
+    await knex.schema.createTable(
+        DatabaseTableName.TECHNICAL_SKILLS,
+        (table) => {
+            table.uuid(DatabaseColumnName.ID).primary();
+            table.string(DatabaseColumnName.SKILL_NAME).notNullable();
+            table.string(DatabaseColumnName.SKILL_LEVEL).notNullable();
```

Comment:
Consider using enum instead of string here

Reply:
Sure, good point)